library/core/src/ops/arith.rs

```diff
@@ -179,12 +179,14 @@ add_impl! { usize u8 u16 u32 u64 u128 isize i8 i16 i32 i64 i128 f16 f32 f64 f128
 /// ```
 #[lang = "sub"]
 #[stable(feature = "rust1", since = "1.0.0")]
+#[rustc_const_unstable(feature = "const_ops", issue = "90080")]
```

Comment:
This an ancient, closed tracking issue. Should we really be using it for new constifications?

I think there should be new tracking issue created, everything else is too confusing.